src/course/handbook/glossary.md

```diff
@@ -0,0 +1,17 @@
+# Glossary or terms
+
+Term | Meaning
+--- | ---
+**argument** | some input to a function
+**attribute** | like a variable, but specifically it's a variable that is defined as part of an *object*
+**function** | a piece of code that takes one or more inputs and returns a value as an output
+**function definition** | The code that defines how a function should behave
+**function call** | The code that makes use of an existing function definition with a specific set of inputs
+**input** | same as *argument*
+**method** | same as a function, but specifically it is a function that is defined as part of an object
+**object** | some code that defines some values (*attributes*) and some functions (*methods*) that are mutually dependent on each other
```

Comment:
This is quite a specific definition of an object.

I think there's a problem for beginners in that JavaScript uses objects both as maps/records/dictionaries (whatever other languages decided to call that data structure) _and_ and a way to group related data/methods.

Beginners mostly encounter objects as a map data structure, and very rarely create their own objects with methods (since OOP is quite uncommon on the web nowadays). If they are creating the OOP-style object they'd probably use a `class`.

A more general (but possibly less useful) definition might be something like: "some code that groups a set of named values together"